lib/elixir/lib/kernel/lexical_tracker.ex

```diff
@@ -46,8 +60,8 @@ defmodule Kernel.LexicalTracker do
 
   # Starts the tracker and returns its pid.
   @doc false
-  def start_link(dest) do
-    :gen_server.start_link(__MODULE__, dest, [])
+  def start_link(dest, parent) do
```

Comment:
Cool. I will ping you soon on IRC. Do not worry about squashing commits for
now, it makes possible reverts like this easier. :)

On Thursday, May 19, 2016, Eric Entin <notifications@github.com> wrote:

> In lib/elixir/lib/kernel/lexical_tracker.ex
> <https://github.com/elixir-lang/elixir/pull/4639#discussion_r63877760>:
>
> > @@ -46,8 +60,8 @@ defmodule Kernel.LexicalTracker do
> >
> >    # Starts the tracker and returns its pid.
> >    @doc false
> > -  def start_link(dest) do
> > -    :gen_server.start_link(__MODULE__, dest, [])
> > +  def start_link(dest, parent) do
>
> Derp. You're totally right, I was just working within the confines of
> each_module and missed this case. I'll back all the "parent" stuff out.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/elixir-lang/elixir/pull/4639/files/26701d64859636f381e5078ac8b0b131a2783afb#r63877760>
>


-- 


*José Valimwww.plataformatec.com.br
<http://www.plataformatec.com.br/>Founder and Director of R&D*